action/protocol/staking/protocol.go

```diff
@@ -471,11 +471,13 @@ func (p *Protocol) ActiveCandidates(ctx context.Context, sr protocol.StateReader
 	if err != nil {
 		return nil, errors.Wrap(err, "failed to get ActiveCandidates")
 	}
-
+	featureCtx, ok := protocol.GetFeatureCtx(ctx)
```

Comment:
use `MustGetFeatureCtx`

Reply:
During service startup, `ActiveCandidates` is called, and the provided context does not contain the required feature. Using `MustGetFeatureCtx ` will lead to a panic.